sympy/combinatorics/tests/test_homomorphisms.py

```diff
@@ -55,3 +55,33 @@ def test_homomorphism():
     assert T.domain == F
     assert T.codomain == D
     assert T(a*b) == p
+
+def test_isomorphisms():
+    # Trivial Case
+    # FpGroup -> FpGroup
+    F, a, b = free_group("a, b")
+    H = FpGroup(F, [a**3, b**3, (a*b)**2])
+    F, c, d = free_group("c, d")
+    G = FpGroup(F, [c**3, d**3, (c*d)**2])
+    assert is_isomorphism(G, H)
+
+    # FpGroup -> PermutationGroup
+    # FpGroup is converted to the equivalent isomorphic group.
+    F, a, b = free_group("a, b")
+    G = FpGroup(F, [a**3, b**3, (a*b)**2])
+    H = AlternatingGroup(4)
+    T = group_isomorphism(G, H)
+    assert T.kernel().order() == G.order()
+    assert T.is_trivial()
+    assert is_isomorphism(G, H)
```

Comment:
If `T` is the trivial homomorphism, then it is definitely not an isomorphism - isomorphisms are injective maps.

Reply:
Yes, sorry about that. The problem is with the homomorphism object returned. I'll change update it.